lib/unison-prelude/src/Unison/Debug.hs

```diff
@@ -163,12 +165,12 @@ debugM flag msg a =
 debugLog :: DebugFlag -> String -> a -> a
 debugLog flag msg =
   if shouldDebug flag
-    then pTrace msg
+    then trace msg
```

Comment:
Calling `pTrace` on something that's already a string, tries to parse it like Show output and ends up hiding bits and pieces of it randomly :'(